packages/type-storage/src/fromMetadata/getHasher.ts

```diff
@@ -2,19 +2,39 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
+import { Text } from '@polkadot/types';
 import { StorageHasher } from '@polkadot/types/primitive';
 import { blake2AsU8a, xxhashAsU8a } from '@polkadot/util-crypto';
 
 type HasherInput = string | Buffer | Uint8Array;
 
-type HasherFunction = (data: HasherInput) => Uint8Array;
+export type HasherFunction = (data: HasherInput) => Uint8Array;
+
+function toStorageHasher (text: Text): StorageHasher {
+  switch (text.toString()) {
+    case 'blake2_128':
+      return new StorageHasher('Blake2_128');
+    case 'blake2_256':
+      return new StorageHasher('Blake2_256');
+    case 'twox_128':
+      return new StorageHasher('Twox128');
+    case 'twox_256':
+      return new StorageHasher('Twox256');
+    default:
```

Comment:
Could you add a `twox_64_concat` case here too?

Reply:
okay

Reply:
Yep. Btw, in StorageHasher, I only see 'Twox128Concat', does 'twox_64_concat' match the 'Twox128Concat'?

Reply:
Actually I think I was mistaken, iirc at some point there was Twox128Concat in the substrate code, but now everything is Twox64Concat.

So in api, we should only have Twox64Concat and twox_64_concat

Reply:
Cool. I see. Thanks @amaurymartiny !